benchmark/Benchmark.h

```diff
@@ -163,10 +251,10 @@ class BenchmarkRegister {
      *
      * @param benchmarks The functions can be passed as
      *  `{&functionName1, &functionname2, ...}`. For more information about
-     *  their usage see _registerdBenchmarks . 
+     *  their usage see getRegister . 
      */
     BenchmarkRegister(const std::vector<std::function<void(BenchmarkRecords*)>>& benchmarks);
 
-    // Return a const view of all registerd benchmarks.
-    static const std::vector<std::function<void(BenchmarkRecords*)>>& getRegisterdBenchmarks();
+    // Return a const view of all registered benchmarks.
+    static const std::vector<std::function<void(BenchmarkRecords*)>>& getRegisteredBenchmarks();
 };
```

Comment:
Don't you want a function `runAllRegisteredBenchmarks()` that creates a new `BenchmarkRecords` object, fills it, and then returns it ? (This functionality is currently probaby in your main function, but main functions should be as small as possible.

Reply:
Done, see [ae26b3f](https://github.com/ad-freiburg/qlever/pull/860/commits/ae26b3fa9c932553629b527452f82739c089c6ce).